java/src/hu/holyoil/controller/AIController.java

```diff
@@ -30,6 +32,10 @@
      * Az pályán található összes teleporter.
      */
     private List<TeleportGate> teleporters;
+    /**
+     * Az AI viselkedéséhez hozzájárló random tényező.
+     */
+    private Random random = new Random();
```

Comment:
We should rather instantiate Random where it's needed, so that the seed is not the same.

Reply:
done